excalidraw/excalidraw
# feat: add custom color picker for canvas background

## Description

This PR adds a custom color picker component for canvas background color selection, addressing issue #8633. The new implementation provides a more intuitive and user-friendly way to select colors while maintaining existing functionality.

### Changes

- Added visual color picker interface with gradient selection
- Implemented live color preview
- Preserved existing eyedropper functionality
- Maintained keyboard navigation and accessibility features

### Screenshots

Before:
![#1](https://github.com/user-attachments/assets/e5a0c91f-584f-4c07-b592-17c125db261f)
After
![#0](https://github.com/user-attachments/assets/d3831a4a-4043-4c14-899a-b29f395116de)

### How to test
1. Open Excalidraw
2. Click to open Hamburger menu
3. Go to Canvas background section
4. Test the new color picker interface
5. Verify eyedropper still works
6. Test keyboard navigation

### Related Issue
Closes #8633

### Type of change
